src/server/bg_services/semaphore_monitor.js

```diff
@@ -52,7 +52,6 @@ class SemaphoreMonitor {
 
     sample_object_io_semaphore() {
         if (!this.object_io) {
-            dbg.log0('semaphore_monitor: object_io is invalid', this.object_io);
```

Comment:
I handled this in my PR - https://github.com/noobaa/noobaa-core/pull/7505/files#diff-88c398d85c1c11beb27354e5271db1dfe0f4b8f94d6050e23f4716c039f68d54
I didn't remove because we need it on containerized installations of noobaa